internal/ui/inbox_test.go

```
@@ -0,0 +1,608 @@
+}
+
+// settledPane waits for the pane to hold every marker and stop changing,
+// since the tool echoes what was typed and then prints it back.
+func settledPane(t *testing.T, m *Model, sessionID string, markers ...string) string {
+	t.Helper()
+	// Two waits, not one. A paste still landing resets the quiet run, so
+	// requiring the markers and the quiet in the same capture can burn
+	// the whole deadline on a loaded runner: first wait for every marker
+	// to have rendered, then for the pane to stop changing.
+	deadline := time.Now().Add(10 * time.Second)
+	previous := ""
```

golangci-lint (2.12.2)

[error] assigned to previous, but reassigned without using the value

(wastedassign)